pyproject.toml

```diff
@@ -45,33 +46,106 @@ exclude = [
+    'datasource/data_connector/configured_asset_azure_data_connector\.py',  # 2
+    'datasource/data_connector/configured_asset_file_path_data_connector\.py',  # 7
+    'datasource/data_connector/configured_asset_sql_data_connector\.py',  # 7
+    'datasource/data_connector/data_connector\.py',  # 4
+    'datasource/data_connector/file_path_data_connector\.py',  # 6
+    'datasource/data_connector/inferred_asset_azure_data_connector\.py',  # 2
+    'datasource/data_connector/inferred_asset_file_path_data_connector\.py',  # 1
+    'datasource/data_connector/inferred_asset_sql_data_connector\.py',  # 4
+    'datasource/data_connector/runtime_data_connector\.py',  # 15
+    'datasource/data_connector/sorter/sorter\.py',  # 1
     'dataset/sparkdf_dataset\.py', # 3
     'dataset/sqlalchemy_dataset\.py', # 16
-    'execution_engine/split_and_sample',  # 65
+    'execution_engine/split_and_sample/data_splitter\.py',  # 6
+    'execution_engine/split_and_sample/data_sampler\.py',  # 1
+    'execution_engine/split_and_sample/pandas_data_sampler\.py',  # 16
+    'execution_engine/split_and_sample/pandas_data_splitter\.py',  # 2
+    'execution_engine/split_and_sample/sparkdf_data_sampler\.py',  # 11
+    'execution_engine/split_and_sample/sparkdf_data_splitter\.py',  # 4
+    'execution_engine/split_and_sample/sqlalchemy_data_sampler\.py',  # 10
+    'execution_engine/split_and_sample/sqlalchemy_data_splitter\.py',  # 22
     'execution_engine/pandas_execution_engine\.py',  # 9
     'execution_engine/sparkdf_execution_engine\.py',  # 8
-    'execution_engine/sqlalchemy_dialect.py', # 1
+    'execution_engine/sqlalchemy_dialect.py',  # 1
     'execution_engine/sqlalchemy_execution_engine\.py',  # 8
     'execution_engine/sqlalchemy_batch_data\.py',  # 1
     'execution_engine/execution_engine\.py',  # 7
-    'render/renderer', # 78
+    'render/renderer/checkpoint_new_notebook_renderer\.py',  # 9
+    'render/renderer/column_section_renderer\.py',  # 1
+    'render/renderer/content_block/bullet_list_content_block\.py',  # 1
+    'render/renderer/content_block/content_block\.py',  # 5
+    'render/renderer/content_block/exception_list_content_block\.py',  # 4
+    'render/renderer/content_block/validation_results_table_content_block\.py',  # 2
+    'render/renderer/datasource_new_notebook_renderer\.py',  #  4
+    'render/renderer/inline_renderer\.py',  # 3
+    'render/renderer/notebook_renderer\.py',  # 2
+    'render/renderer/page_renderer\.py',  # 10
+    'render/renderer/profiling_results_overview_section_renderer\.py',  # 2
+    'render/renderer/site_builder\.py',  # 3
+    'render/renderer/suite_edit_notebook_renderer\.py',  # 7
+    'render/renderer/suite_scaffold_notebook_renderer\.py',  # 7
+    'render/renderer/v3/suite_edit_notebook_renderer\.py',  # 11
+    'render/renderer/v3/suite_profile_notebook_renderer\.py',  # 4
     'render/view',  # 11
     'render/types',  # 4
-    'expectations/expectation', # 188
+    'expectations/core',  # 257
```

Comment:
This is likely much higher because the previously type-check did not install needed type-stubs (which expose additional issues)